src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -8302,10 +8155,10 @@ Real64 CalcGoldsteinNovoselacCeilingDiffuserWindow(EnergyPlusData &state,
             ShowContinueError(state, "Occurs for zone named = " + Zone(ZoneNum).Name);
             ShowContinueError(state, "Convection surface heat transfer coefficient set to 9.999 [W/m2-K] and the simulation continues");
         }
-        ShowRecurringSevereErrorAtEnd(
-            state,
-            "CalcGoldsteinNovoselacCeilingDiffuserWindow: Convection model not evaluated because bad perimeter length and set to 9.999 [W/m2-K]",
-            state.dataConvectionCoefficient->CalcGoldsteinNovoselacCeilingDiffuserWindowErrorIDX2);
+        ShowRecurringSevereErrorAtEnd(state,
+                                      "CalcGoldsteinNovoselacCeilingDiffuserWindow: Convection model not evaluated because bad perimeter "
+                                      "length and set to 9.999 [W/m2-K]",
+                                      state.dataConvectionCoefficient->CalcGoldsteinNovoselacCeilingDiffuserWindowErrorIDX2);
```

Comment:
I didn't realize I introduce this change. I think I may have a newer clang format or this file wasn't clang formatted for a while. Sorry... I could manually revert these changes if it's a problem @Myoldmopar